packages/world/test/BiomesAssertions.sol

```diff
@@ -88,8 +88,8 @@ abstract contract BiomesAssertions is MudTest, GasReporter {
   function assertEnergyFlowedFromPlayerToLocalPool(
```

Comment:
would it be cleaner to instead add the expected loss as an argument instead? and assertLt() inside?

Reply:
but then for each test we'd have to calculate the expected loss, right now I just want _some_ loss, dont care about the amount

Reply:
what about defining an overloaded function with the same name? That way you don't have to duplicate the check everywhere

Reply:
> That way you don't have to duplicate the check everywhere
wdym?

Reply:
have an additional assertEnergyFlowedFromPlayerToLocalPool that receives an extra parameter and does the require internally

Reply:
ahh okay will consider in future PR